docs/.vuepress/config.js

```diff
@@ -26,7 +26,20 @@ const sidebar = {
       children: [
         ['/developer-docs/latest/development/backend-customization.md', 'Backend customization'],
         ['/developer-docs/latest/development/admin-customization.md', 'Admin panel customization'],
-        ['/developer-docs/latest/development/plugin-customization.md', 'Strapi plugins'],
+        {
+          title: 'Strapi plugins',
+          path: '/developer-docs/latest/development/plugin-customization.html',
+          collapsable: true,
+          children: [
+            ['/developer-docs/latest/development/plugins/documentation.md', 'API Documentation'],
+            ['/developer-docs/latest/development/plugins/email.md', 'Email'],
+            ['/developer-docs/latest/development/plugins/graphql.md', 'GraphQL'],
+            ['/developer-docs/latest/development/plugins/i18n.md', 'i18n'],
```

Comment:
```suggestion
['/developer-docs/latest/development/plugins/i18n.md', 'Internationalization (I18N)'],
```

Reply:
Or

```suggestion
['/developer-docs/latest/development/plugins/i18n.md', 'I18N'],
```

Reply:
Thanks Derrick! 
I'll add the full word.
I don't think we should capitalize the 'I' and 'N', though. I18N → i18n

Reply:
numeronyms are usually capital based but in searching it appears i18n is one of those odd ball ones that's not.